modules/ircd/irc.lineaire.nix

```diff
@@ -1,16 +1,18 @@
 { config, lib, ... }: let
-  acmeDirectory = config.security.acme.certs."lineaire.moe".directory;
+  domain = "lineaire.moe";
+  acmeDirectory = config.security.acme.certs."${domain}".directory;
 in
   {
+  networking.domains.subDomains.${domain} = { };
```

Comment:
what does this line do?

Reply:
the base domain in `hosts/lina.nix` propagates it's records to any subdomain matching "lineaire.moe" or "*.lineaire.moe" so it gets:
```nix
        aaaa.data = "2001:bc8:38ee:100:500::1";
```

as automatic record + (the a records specified in that line in the new patchset)

this is kind of the "magic sauce" of nixos-dns. for example if you just put `networking.domains.subDomains."prometheus.${config.networking.fqdn}" = { };` and import it to any host then you can auto-magically generate all the records for exporters you want which is kinda neat.